languages.json

```diff
@@ -79,6 +80,7 @@
     {"lang": "Jade", "scopes": ["source.css", "source.jade"], "files": ["jade"]},
     {"lang": "Ruby Slim", "scopes": ["source.css", "text.slim"], "files": ["slim"]},
     {"lang": "Less", "scopes": ["source.css", "source.less"], "files": ["less"]},
+    {"lang": "PostCSS", "scopes": ["source.css", "source.pcss", "source.postcss"], "files": ["pcss", "postcss"]},
```

Comment:
Binding `source.css` to PostCSS is problematic as it might override the syntax highlighting if you are using a plain CSS file, at least this is my concern.

Can you remove it?